woo-razorpay.php

```diff
@@ -270,29 +271,35 @@ public function __construct($hooks = true)
             // 1cc flags should be enabled only if merchant has access to 1cc feature
             $is1ccAvailable = false;
             $isAccCreationAvailable = false;
+            $merchantPreferences = [];
 
-            // Load preference API call only for administrative interface page.
-            if (current_user_can('administrator'))
+            $merchantPreferences = get_transient(self::ONE_CC_MERCHANT_PREF);
+
+            // Load preference API call only for administrative interface + razorpay payment settings page.
+            if (current_user_can('administrator') &&
```

Comment:
@ChetanGN , any analysis why too many API hits , though we have admin check here ?